pkg/util/util.go

```diff
@@ -1479,3 +1479,17 @@ func GetEnvWithDefault(key string, defaultval string) string {
 	}
 	return val
 }
+
+//IsValidKubeConfigPath checks if specified `KUBECONFIG` value is valid i.e it returs true if `KUBECONFIG` env is not set
```

Comment:
I think it could be interesting to handle the case where KUBECONFIG is not defined, but ~/.kube/config does not exist

Reply:
I wonder how safe it would be though. AFAIK the default kubeconfig is created by clients if it does not exist. Correct me if i am wrong?

Reply:
Clients are editing/creating the kubeconfig file depending on the KUBECONFIG (or ~/.kube/config by default). I can easily imagine a case where I ran `KUBECONFIG=$HOME/.kube/config-crc crc start` (which will create/edit `~/.kube/config-crc`), and then run `odo catalog list components` (without KUBECONFIG defined, and no `~/.kube/config` file)

Reply:
ok lemme update the check

Reply:
Please also update the comment